modules/context-based-restrictions/pom.xml

```
@@ -0,0 +1,68 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+	<!-- >>> Replace this with the parent pom's artifactId -->
+        <artifactId>platform-services</artifactId>
+        <groupId>com.ibm.cloud</groupId>
+        <version>99-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <!-- >>> Replace this with the service module's artifactId (e.g. "example-service") -->
+    <artifactId>context-based-restrictions</artifactId>
+
+    <!-- >>> Replace this with a text description of this module (e.g. "Example Service") -->
+    <name>Context-Based Restrictions</name>
```

```suggestion
    <name>IBM Cloud Context-Based Restrictions API</name>
```
(just to be consistent with the other modules)